plugins/crd/datastore/report_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datastore
+
+import (
+	"github.com/ligato/cn-infra/logging/logrus"
+	"testing"
+	"time"
+)
+
+func TestSimpleReport_AppendToNodeReport(t *testing.T) {
+	report := NewSimpleReport(logrus.DefaultLogger())
+	report.LogErrAndAppendToNodeReport("nodeName", "ErrorString")
```

Comment:
here check that the length of log in lw is 1 
also, check that length of report.Data[] is 1